src/components/dashboard/FirstSteps/index.tsx

```diff
@@ -0,0 +1,236 @@
+              Add funds
+            </Button>
+          </Box>
+          <ModalDialog
+            open={open}
+            onClose={toggleDialog}
+            dialogTitle="Add funds to your Safe Account"
+            hideChainIndicator
+          >
+            <Box px={4} pb={5} pt={4}>
+              <Grid container spacing={2} alignItems="center" mb={4}>
+                <Grid item>
+                  <Box p={1} border={1} borderRadius="6px" borderColor="border.light" display="inline-flex">
+                    <QRCode value={safeAddress} size={132} />
+                  </Box>
+                </Grid>
+                <Grid item xs>
+                  <Typography mb={2}>
+                    Add funds directly from your bank account or copy your address to send tokens from a different
+                    account.
+                  </Typography>
+
+                  <Box bgcolor="background.main" p={2} borderRadius="6px" alignSelf="flex-start" fontSize="14px">
+                    <EthHashInfo
+                      address={safeAddress}
+                      showName={false}
+                      shortAddress={true}
+                      showCopyButton
+                      hasExplorer
+                      avatarSize={24}
+                    />
+                  </Box>
+                </Grid>
+              </Grid>
+
+              <Box mb={4} position="relative" textAlign="center">
+                <Typography className={css.orDivider}>or</Typography>
+                <Divider />
+              </Box>
+
+              <Typography mb={2}>Buy crypto with fiat:</Typography>
+              <BuyCryptoOptions />
+            </Box>
+          </ModalDialog>
+        </>
+      )}
+    </StatusCard>
+  )
+}
+
+const FirstTransactionWidget = ({ completed }: { completed: boolean }) => {
+  const [open, setOpen] = useState<boolean>(false)
+
+  const title = 'Create your first transaction'
+  const content = 'Simply send funds, add a new signer or swap tokens through a safe app.'
+
+  return (
+    <>
+      <StatusCard badge="First interaction" title={title} content={content} completed={completed}>
+        {!completed && (
+          <Button onClick={() => setOpen(true)} variant="outlined" size="small" sx={{ mt: 2, minHeight: '40px' }}>
+            Create transaction
+          </Button>
+        )}
+      </StatusCard>
+      <FirstTxFlow open={open} onClose={() => setOpen(false)} />
+    </>
+  )
+}
+
+const AccountReadyWidget = () => {
+  return (
+    <Card className={classnames(css.card, css.accountReady)}>
+      <div className={classnames(css.checkIcon)}>
+        <CheckCircleOutlineRoundedIcon sx={{ width: '60px', height: '60px' }} />
+      </div>
+      <Typography variant="h4" fontWeight="bold" mb={2} mt={2}>
+        Safe Account is ready!
+      </Typography>
+      <Typography>Continue to improve your account security and unlock more features</Typography>
+    </Card>
+  )
+}
+
+const FirstSteps = () => {
```

Comment:
FirstSteps is a very broad name. Perhaps OnboardingSafeActions or something like that?